<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>61</y>
      <width>322</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sätt antal steg för miskning/ökning av reglagen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderRed">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>196</y>
      <width>16</width>
      <height>84</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRed">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>164</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Red">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>141</y>
      <width>27</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>55</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_Green">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxGreen">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSliderGreen">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>55</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_Blue">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxBlue">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="verticalSliderBlue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxRed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderRed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxGreen</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderGreen</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxBlue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderBlue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
